service/src/jestSetup/e2eFrameworkSetup.js

```diff
@@ -9,6 +9,19 @@ jasmine.DEFAULT_TIMEOUT_INTERVAL = 5000 //eslint-disable-line
 
 beforeEach(async () => {
     global.page = await global.browser.newPage()
+    try {
+        const testNames = Object.keys(global.globalContext.testNames).join(', ')
```

Comment:
```suggestion
        const testNames = Object.keys(global.globalContext.testNames).sort().join(', ')
```

for some determinism.

Also this could potentially grow quite large meaning it might not be appropriate for an HTTP header?

Reply:
will be small now thats its only 1 test . grabbing name via getState from jest